sdk/include/psdk/winuser.h

```diff
@@ -1221,6 +1221,7 @@ extern "C" {
 #define SIF_RANGE 1
 #define SIF_DISABLENOSCROLL 8
 #define SIF_TRACKPOS   16
+#define SIF_THEMED 128      /* REACTOS Specific Only */
```

Comment:
So, how windows is able to manage that if it doesn't need such flag?

Reply:
> So, how windows is able to manage that if it doesn't need such flag?

Sorry, I do not know and I am not able to use WinDbg enough to find out. I wish that I was able to answer your question.

Reply:
Then maybe it'd be the opportunity for you right now to start learning how to use WinDBG?

Reply:
Most likely the window properties I commented about: https://jira.reactos.org/browse/CORE-16735